packages/@lwc/synthetic-shadow/src/faux-shadow/shadow-root.ts

```diff
@@ -271,7 +267,7 @@ const NodePatchDescriptors = {
             newChild: T,
             refChild: Node | null
         ): T {
-            insertBefore.call(getHost(this), newChild, refChild);
+            Node.prototype.insertBefore.call(getHost(this), newChild, refChild);
```

Comment:
I'm worry that we are moving from the cached node method to a non-cached version. any particular reason?

Reply:
This forces synthetic-shadow to use the patches applied in node-reaction. We could not use the cached methods because synthetic-shadow is applied before node-reactions initialization. This is because node-reactions relies on some polyfills of synthetic-shadow(ShadowRoot).